src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
+  font-family: avenir, sans-serif;
+  font-style: normal;
+  margin-bottom: 2%;
+  margin-top: 20px;
+  padding-left: 10%;
+}
+div.red_bar {
+  width: 70px;
+  margin-bottom: 2%;
+  border-top: 7px solid #df3143;
+  margin-left: 10%;
+}
+
+section#quick_id {
+  float: left;
+  color: white;
+  background-color: #2e4975;
+  width: 50%;
+  height: 300px;
+}
+section#contact_me {
+  float: right;
+  background-color: lightsteelblue;
+  width: 50%;
+  height: 300px;
+}
+
+table {
+  padding-left: 10%;
+  text-align: left;
+  font-size: 20px;
+  font-family: sans-serif;
+}
+table td {
+  padding-left: 20%;
+  font-weight: 200;
+}
+table th {
+  text-align: start;
+  text-combine-upright: none;
+  vertical-align: top;
+  font-weight: 700;
+}
+form {
+  padding-left: 5%;
+}
+form label {
+  display: none;
+}
+form input {
+  background-color: lightsteelblue;
+  color: white;
+  border: 2px hidden lightsteelblue;
```

Comment:
should be 1px and wrong color.

Reply:
and what is `hidden`? could you point out the reference if you are using an rare property and at least check https://caniuse.com

Reply:
should be just `border: 1px solid transparent;`